decidim-elections/app/models/decidim/votings/polling_station.rb

```diff
@@ -6,10 +6,55 @@ module Votings
     class PollingStation < ApplicationRecord
       include Traceable
       include Loggable
+      include Decidim::TranslatableResource
+
+      translatable_fields :title, :location, :location_hints
 
       belongs_to :voting, foreign_key: "decidim_votings_voting_id", class_name: "Decidim::Votings::Voting", inverse_of: :polling_stations
+      has_many :polling_station_managers,
+               foreign_key: "managed_polling_station_id",
+               class_name: "Decidim::Votings::PollingOfficer",
+               inverse_of: :managed_polling_station,
+               dependent: :nullify
+      has_one :polling_station_president,
+              foreign_key: "presided_polling_station_id",
+              class_name: "Decidim::Votings::PollingOfficer",
+              inverse_of: :presided_polling_station,
+              dependent: :nullify
+
+      validate :polling_station_managers_same_voting
+      validate :polling_station_president_same_voting
+
+      # Allow ransacker to search for a key in a hstore column (`title`.`en`)
+      ransacker :title do |parent|
+        Arel::Nodes::InfixOperation.new("->>", parent.table[:title], Arel::Nodes.build_quoted(I18n.locale.to_s))
+      end
+
+      [:manager, :president].each do |role|
+        [:name, :email, :nickname].each do |field|
+          ransacker "#{role}_#{field}".to_sym do
+            Arel.sql("#{role}_user.#{field}")
+          end
+        end
+      end
```

Comment:
Smart! 🤓

Reply:
Wanted to use `[:manager, :president].product([:name, :email, :nickname])` but then it would have been unreadable 😓